api/astro-date.js

```
@@ -0,0 +1,154 @@
+    const signs = Object.entries(positions).reduce(
+      /**
+       * @param {Record<string, string>} acc
+       * @param {Array} entry - [planet, position] tuple
+       * @returns {Record<string, string>} Updated accumulator
+       */
+      (acc, entry) => {
+        const planet = entry[0];
+        /** @type {PlanetPosition} */
+        const position = entry[1];
+        return {
+          ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)